Woof/Woof/View/ReusableViews/LabelWithTextAndImage.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+
+/// A view displaying an icon and the portion of text.
```

Comment:
```suggestion
/// A view displaying an icon and the portion of text (what size? what colour?).
```
Does the view itself has a pre-defined colour?
If so, we can update the name as well. Smth like:
```swift
PurpleLabelWithTextAndImage
```

Reply:
Thanks for the useful comment! 
After discussion with the team, we decided that we don't need this reusable element.
So the file `LabelWithTextAndImage.swift` are deleted.